src/Message/MessageParser.ts

```diff
@@ -82,14 +84,15 @@ const tagUnescapeMap: { [char: string]: string } = {
 	s: ' '
 };
 
-export function parseTags(raw: string): Map<string, string> {
+export function parseTags(raw: string): Map<string, string> | undefined {
 	const tags: Map<string, string> = new Map();
 	const tagStrings = raw.split(';');
 	for (const tagString of tagStrings) {
 		const [tagName, tagValue] = splitWithLimit(tagString, '=', 2);
 		// unescape according to http://ircv3.net/specs/core/message-tags-3.2.html#escaping-values
+		if (tagName === '') continue; // Ignore empty tags: @ @; @x; etc.
```

Comment:
While this change makes sense, its placement is a little bit off. It breaks the context of the comment above it, which is supposed to be above the `tags.set()` line below.

Also, a condition body goes in braces. Please run the linter (`yarn lint` or `npm run lint`) before committing.

Reply:
Tbh, I edited on Github website. I usually put it in braces, not sure why I didn't this time. I'll fix that now.